onnx/backend/test/runner/__init__.py

```diff
@@ -232,21 +232,21 @@ def download_model(
         # On Windows, NamedTemporaryFile can not be opened for a
         # second time
         del model_dir
-        download_file = tempfile.NamedTemporaryFile(delete=False)
```

Comment:
with delete being false, do we still need to use it as a context manager?

Reply:
To make ruff happy now